content/departments/technical-success/ce/process/license_keys.md

```diff
@@ -40,8 +40,13 @@ The CE should first create a Sourcegraph.com user account for the prospect/custo
 2. If the customer does not have a license yet, navigate to the [users page](https://sourcegraph.com/site-admin/users) and create new user account. (You can also check for a user already exists following the instructions in the [Internal Licensing FAQ](#internal-licensing-FAQ) document.)
 3. Create an account with the username `CompanyName-UniqueId`. Replace `CompanyName` with the company name, and use the Unique ID from the Unique Account ID field on the Account record in Salesforce.
 4. Navigate to the [subscriptions page](https://sourcegraph.com/site-admin/dotcom/product/subscriptions). For new prospects during trial and / or a new customer, click **Create product subscription**. Search for the user you just created.
-5. Click **Generate new license manually**. Fill out the license end date (most typically to match the contract terms) and fill in the appropriate license tags. For tags, see [License Key Tags](#license-key-tags) for a list you can just copy. Remember: Tags must be separated by commas, with no spaces!
-6. Set the licensed number of users (note that if you added the `true-up` tag, the company will be able to exceed this count, but administrators will see a warning) and the number of days that the license should be valid, and click **Generate license**.
+5. Click **Generate new license manually**.
+   1. Fill out the customer name, lowercase, fill spaces with `-` character
+   1. Fill out the license subscription plan name [see the plans](#plans) section
+   1. Fill out the licensed number of users. (Note that if you added the `true-up` tag, the company will be able to exceed this count, but administrators will see a warning).
+   1. Fill out the number of days the license should be valid (most typically to match the end date of the contract itself)
+   1. Fill in the appropriate license tags. For tags, see [License Key Tags](#license-key-tags) section. Remember: Tags must be separated by commas. You can see a list of tags generated under the tags input. If tag is not recognized, it will be red and a warning will appear.
+6. Click **Generate license**.
```

Comment:
```suggestion
   1. Fill out the customer name in lowercase. Fill spaces with the `-` character.
   1. Fill out the license subscription plan name. See the [plans](#plans) section.
   1. Fill out the licensed number of users. Note that if you added the `true-up` tag, the customer will be able to exceed this count, but administrators will see a warning.
   1. Fill out the number of days the license should be valid for. Most typically this should match the end date of the contract itself.
   1. Fill in the appropriate license tags. For tags, see the [License Key Tags](#license-key-tags) section. Remember: Tags must be separated by commas. You can see a list of tags generated under the tags input. If a tag is not recognized, it will be red and a warning will appear.
6. Click **Generate license**.
```